src/logger.ts

```diff
@@ -56,7 +56,7 @@ export function write(entry: LogEntry) {
 
 /**
  * Writes a `DEBUG` severity log. If the last argument provided is a plain object,
- * it will be added to the `jsonPayload` in the Cloud Logging entry.
+ * it is added to the `jsonPayload` in the Cloud Logging entry.
```

Comment:
Here's a link to an explanation of jsonPayload: https://cloud.google.com/logging/docs/structured-logging

Not sure if that's TMI though

If added, this applies to all comments that mention jsonPayload

Reply:
Hm, that's a good reference for the guide material, as well as for a top-level comment for `logger` -- which I should probably add to the main page here.